dts/bindings/mmc/nxp,imx-usdhc.yaml

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2019, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: NXP i.MXRT USDHC module
+version: 0.1
+
+description: >
+    This binding specifies the NXP i.MXRT USDHC module.
+
+inherits:
+    !include mmc.yaml
+
+properties:
+    compatible:
+      constraint: "nxp,imx-usdhc"
+
+    clocks:
+      category: required
+
+    pwr-gpios:
+      type: compound
+      category: optional
+      description: Power pin
+      generation: define, use-prop-name
+
+    cd-gpios:
+      type: compound
+      category: optional
+      description: Detect pin
+      generation: define, use-prop-name
```

Comment:
Remove both instances of `use-prop-name`

Reply:
fixed